site/_includes/icon-discord.html

```diff
@@ -0,0 +1,4 @@
+<a href="https://https://discord.com/invite/{{ include.username }}">
```

Comment:
Should this have a single `https://` ?

```suggestion
<a href="https://discord.com/invite/{{ include.username }}">
```